controller/Controller.class.php

```diff
@@ -153,6 +153,53 @@ protected static function getRouterWithRoutes(): \Routing\RouteCollector
       '/faq/Q1-17-CreditReport'             => '/credit-reports/2017-Q1',
       '/faq/how-to-report-bugs'             => '/faq/support',
       '/faq/make-money'                     => '/faq/earn-income',
+
+//     New Redirects
+      
+      '/news/$1.2b-market-cap-we-dont-care' => '/news/1.2b-market-cap-we-dont-care',      
```

Comment:
What if we move all of this to the 404 route / handler so that they don't have to be declared as routes?

Reply:
@kauffj who can work on this? @maximest-pierre ?